docs/web/index.md

```diff
@@ -539,3 +540,55 @@ Logs are printed to standard out.
+continue here:
+
+Services that use the sda-spring-boot-commons:
+- can access the current [MetadataContext](../../sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/metadata/MetadataContext.java)
+  in their implementation
+- will automatically load the context from incoming HTTP requests into the thread handling the
+  request, if you register [MetadataContextConfiguration](../../sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/metadata/MetadataContextConfiguration.java)
+- will automatically load the context from consumed Kafka messages into the thread handling the
+  message and the error when handling the message fails when the consumer is configured with one of
+  the
+  provided [SdaKafkaConsumerConfiguration](../../sda-commons-starter-kafka/src/main/java/org/sdase/commons/spring/boot/kafka/SdaKafkaConsumerConfiguration.java)
+- will automatically propagate the context to other services via HTTP when using a platform client
+  that uses
+  the [MetadataContextClientConfiguration](../../sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/metadata/MetadataContextConfiguration.java)
+  configuration, e.g:
+  - ```java
+    @FeignClient(
+    value = "name",
+    url = "http://your-api-url",
+    configuration = {
+      MetadataContextClientConfiguration.class
+    })
+    public interface ClientWithMetadataConfiguration {
+
+    @GetMapping("/metadata-hello")
+    Object getSomething();
+    }
+    ```
+- will automatically propagate the context in produced Kafka messages when the producer is created
+  with [SdaKafkaProducerConfiguration](../../sda-commons-starter-kafka/src/main/java/org/sdase/commons/spring/boot/kafka/SdaKafkaProducerConfiguration.java)
+- are configurable by the property or environment variable `METADATA_FIELDS` to be aware of the
+  metadata used in a specific environment
+
+Services that interrupt a business process should persist the context from
+`MetadataContext.detachedCurrent()` and restore it with `MetadataContext.createContext(…)` when the
+process continues.
+Interrupting a business process means that processing is stopped and continued later in a new thread
+or even another instance of the service.
+Most likely, this will happen when a business entity is stored based on a request and loaded later
+for further processing by a scheduler or due to a new user interaction.
+In this case, the `DetachedMetadataContext` must be persisted along with the entity and recreated
+when the entity is loaded.
+The `DetachedMetadataContext` can be defined as field in any MongoDB entity.
+
+Services that handle requests or messages in parallel must transfer the metadata context to their
+`Runnable` or `Callable` with `MetadataContext.transferMetadataContext(…)`.
+In most cases, developers should prefer`ContainerRequestContextHolder.transferRequestContext(…)`,
+which also transfers the metadata context.
```

Comment:
Imho this does not apply for Spring Boot. Or at least is in most cases not required because we should have an implementation like [this](https://github.com/SDA-SE/sda-spring-boot-commons/blob/c20a626fe9135178d7dc7a898937308203829f39/sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/async/ContextCopyTaskDecorator.java#L22) to support `@org.springframework.scheduling.annotation.Async`.

Reply:
Do we have something like that or do we need another task before we can merge the feature?